streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -1575,7 +1576,7 @@ void addTask(final Task task) {
         tasks.addTask(task);
     }
 
-    Tasks tasks() {
+    TasksRegistry tasks() {
```

Comment:
nit: Maybe we want to update the name along with the type?

Reply:
Yeah, I think that is a good idea for this method. But I also think that this method is not needed anymore and it will be removed once the tests in `TaskManagerTest` are updated. I will leave it as it is in this PR and consider the  change in one of the following PRs. Thank you!